packages/ember-glimmer/tests/integration/content-test.js

```diff
@@ -484,6 +484,29 @@ class DynamicContentTest extends RenderingTest {
     this.assertContent('hello');
     this.assertInvariants();
   }
+
+  ['@test it can render a property on a function']() {
+    let func = () => {};
+    func.aProp = 'this is a property on a function';
+
+    this.renderPath('func.aProp', { func });
+
+    this.assertContent('this is a property on a function');
+
+    this.assertStableRerender();
+
+    // this.runTask(() => set(func, 'aProp', 'still a property on a function'));
```

Comment:
The test was trying to do INUR (which is likely what prompted the fixing of the updating side also), so my guess is that the update/reset parts here are just commented out.